src/django_components/component_registry.py

```diff
@@ -42,18 +42,19 @@ def clear(self) -> None:
         self._registry = {}
 
 
-# This variable represents the global component registry
 registry: ComponentRegistry = ComponentRegistry()
+"""Global component registry."""
 
 
 def register(name: str) -> Callable[[_TC], _TC]:
     """Class decorator to register a component.
 
     Usage:
-
-    @register("my_component")
-    class MyComponent(component.Component):
-        ...
+        ```python
```

Comment:
Later, we will need to go over all the docstrings and update them to use the Google dostrings style as set here https://github.com/EmilStenstrom/django-components/pull/420/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R184 (or set it to different style).

Till now, when I was writing docstrings, I formatted it so that it would look good for me on VSCode.